Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml

```diff
@@ -3891,28 +2835,44 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
-  "150":
-    id: "150"
-    taskid: aa54e270-df96-4b84-8046-dd5169e21914
-    type: title
+  "164":
+    id: "164"
+    taskid: 8ffa4256-168f-40ab-8854-5bdebd63711f
+    type: regular
     task:
-      id: aa54e270-df96-4b84-8046-dd5169e21914
+      id: 8ffa4256-168f-40ab-8854-5bdebd63711f
       version: -1
-      name: Prompt No AR
-      type: title
+      name: Set remediation grid field
+      description: |-
+        Automation used to more easily populate a grid field.  This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well.  Instead of a value you can enter `TIMESTAMP` to get the current timestamp in ISO format.  Example of command:
```

Comment:
```suggestion
        Automation used to more easily populate a grid field.  This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well.  Instead of a value you can enter `TIMESTAMP` to get the current timestamp in ISO format.  Example of the command:
```